[HOPSWORKS-1391] Extend Pandas and Numpy helper classes

jimdowling opened the pull request:
https://logicalclocks.atlassian.net/browse/HOPSWORKS-1391

moritzmeister commented on hops/numpy_helper.py:
```diff
@@ -36,9 +125,65 @@ def save(hdfs_filename, data):
+
+    Args:
+       :hdfs_filename: You can specify either a full hdfs pathname or a relative one (relative to your Project's path in HDFS)
+       :args : Arguments, optional
```
Might break the docs otherwise
```suggestion
       :args: Arguments, optional
```

moritzmeister commented on hops/numpy_helper.py:
```diff
@@ -36,9 +125,65 @@ def save(hdfs_filename, data):
+       :args : Arguments, optional
+               Arrays to save to the file. Since it is not possible for Python to know the names of the arrays outside savez, 
+               the arrays will be saved with names “arr_0”, “arr_1”, and so on. These arguments can be any expression.
+       :kwds : Keyword arguments, optional
```
```suggestion
       :kwds: Keyword arguments, optional
```

moritzmeister commented on hops/numpy_helper.py:
```diff
@@ -36,9 +125,65 @@ def save(hdfs_filename, data):
+
+    Args:
+       :hdfs_filename: You can specify either a full hdfs pathname or a relative one (relative to your Project's path in HDFS)
+       :args : Arguments, optional
```
```suggestion
       :args: Arguments, optional
```

jimdowling commented on hops/numpy_helper.py:
```diff
@@ -36,9 +125,65 @@ def save(hdfs_filename, data):
+       :args : Arguments, optional
+               Arrays to save to the file. Since it is not possible for Python to know the names of the arrays outside savez, 
+               the arrays will be saved with names “arr_0”, “arr_1”, and so on. These arguments can be any expression.
+       :kwds : Keyword arguments, optional
```
Ok, thanks. i fixed these.

jimdowling commented:
I have tested this on a VM, added some tests to it_test notebook that work. I removed support for HDF as it requires the tables library (not included in our base env), so they developers can do it themselves if they want to use hdf.
